test/intern.js

```diff
@@ -0,0 +1,35 @@
+
+define({
+  proxyPort: 9000,
+  proxyUrl: "http://localhost:9000/",
+  capabilities: {},
+  environments: [
+    { browserName: "chrome" },
+    { browserName: "internet explorer", version: [ "11", "10" ] },
+    { browserName: "firefox", version: "34" }
```

Comment:
One additional comment: When testing IE8, we need to include ES5 shim (see `unit-es5-shim.html` and `unit-es5-shim.html`).